.Gruntfile.babel.js

```diff
@@ -318,3 +325,38 @@ function script2ify (file) {
     cb(null, buf.toString('utf8').replace(regex, replacement))
   })
 }
+//
+var localeObject = {}
+try {
+  localeObject = JSON.parse(fs.readFileSync('./frontend/simple/locales/en/translation.json'))
+} catch (ex) {
+}
+function localify (file) {
+  return !/\.(vue|html|ejs)$/.test(file)
+    ? through()
+    : through(function (buf, encoding, cb) {
+      let functionEx = new RegExp(/([.|.$])L(\s*)\((\s*)(['|"])(\S*)(['|"])(\s*)([,|)])((\s*)(['|"])(\S*)(['|"])([,|)]))*/mg)
```

Comment:
Oh, this site is also useful for testing regex's against strings: https://regex101.com/